### DISCUSSION TITLE
Searchbar in start menu

### DISCUSSION THREAD

**Discussion by Osahashi** [ramensoftware/windhawk-mods#1285] [Q&A] [answered]
Is there any chance to hide/remove the search bar at top in the start menu?

I'm using the normal Win11 menu with the noRecommendedSection.

**Comment by m417z** [comment, marked as answer]
Yes, check out the Windows 11 start menu styling guide:
https://github.com/ramensoftware/windows-11-start-menu-styling-guide?tab=readme-ov-file#remove-the-search-box
